src/pages/Login.js

```diff
@@ -0,0 +1,85 @@
+/*
+  Copyright 2019 Globo.com authors. All rights reserved.
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file.
+*/
+
+import React, { Component } from 'react';
+import LoginButton from '../components/LoginButton';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGitlab, faGithub } from '@fortawesome/free-brands-svg-icons';
+import logo from './../logo.png';
+
+const colorBackground = '#171515';
+const colorBackgruondBox = '#3d4572';
```

Comment:
Could you fix this typo here, please? `colorBackgruondBox` into `colorBackgroundBox`.

Reply:
Sure!